db/seeds/roles.rb

```diff
@@ -6,7 +6,8 @@
     read_services: true,
     update_services: true,
     delete_services: true,
-    read_appointments: true
+    read_appointments: true,
+    create_appointments: true
```

Comment:
freelancers can create appointments? Should be clients only.

also need to add `update_appointments` for freelancers.